superset-frontend/src/views/CRUD/data/dataset/AddDataset/Footer/Footer.test.tsx

```diff
@@ -20,10 +20,47 @@ import React from 'react';
 import { render, screen } from 'spec/helpers/testing-library';
 import Footer from 'src/views/CRUD/data/dataset/AddDataset/Footer';
 
+const mockedProps = {
+  url: 'realwebsite.com',
+};
+
+const mockPropsWithDataset = {
+  url: 'realwebsite.com',
+  datasetObject: {
+    database: {
+      id: '1',
+      database_name: 'examples',
+    },
+    owners: [1, 2, 3],
+    schema: 'public',
+    dataset_name: 'Untitled',
+    table_name: 'real_info',
+  },
+};
+
 describe('Footer', () => {
-  it('renders a blank state Footer', () => {
-    render(<Footer />);
```

Comment:
It seems like we were rendering this component before without required props, and now, url and addDangerToast are required. Question, are there any other call to this component from before that was not updated to send the new props?

Reply:
This is a good question. This was a test that was created as a placeholder while we were implementing the blank state PR. Now that it is finished we are changing it to be reflective of the actual behavior.